workflows/dags/data/data_harvest.py

```diff
@@ -115,7 +118,8 @@ def add_keywords(record, builder):
                                 f"{keyword}: {item[0]['lte']}-{item[0]['gte']}"
                             )
                     elif keyword == "observables":
-                        builder.add_keyword(f"{keyword}: {','.join(item)}")
+                        for value in item:
+                            builder.add_keyword(f"observables: {value}")
```

Comment:
@michamos do we need to add this prefix?

Reply:
yes, is it a problem?

Reply:
No, I’m just thinking it’s a bit dirty for metadata. Can we add a `type` for keywords? Or did I misunderstand the entire thing?

Reply:
that's how we encode hierarchical keywords on INSPIRE, see https://inspirehep.net/literature/1998859 for example